signal/client/grpc.go

```diff
@@ -71,7 +72,7 @@ func NewClient(ctx context.Context, addr string, key wgtypes.Key, tlsEnabled boo
 		transportOption = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{}))
 	}
 
-	sigCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	sigCtx, cancel := context.WithTimeout(ctx, client.ConnectTimeout)
```

Comment:
let's replace the `defaultTimeout` on line 357 to use the `client.Connecttimeout`

Reply:
Addressed in 780983eadf1fd7f209fa01ff070d0993aa80ec41.